python_modules/libraries/dagster-azure/dagster_azure/adls2/file_manager.py

```diff
@@ -14,10 +15,11 @@
 class ADLS2FileHandle(FileHandle):
     """A reference to a file on ADLS2."""
 
-    def __init__(self, account: str, file_system: str, key: str):
+    def __init__(self, account: str, file_system: str, key: str, primary_endpoint: str):
```

Comment:
[1] Could we default to `None` here and handle legacy code?  

Otherwise, I believe this will break current usage of `ADLS2FileManager.read` and `ADLS2FileManager.read_data`

Reply:
edited to default to None and handle legacy code